packages/tools/stylelint-config-pie/index.js

```diff
@@ -1,6 +1,7 @@
-import baseRules from './rules/base';
-import strictRules from './rules/strict';
-import orderingRules from './rules/ordering';
+/* eslint-disable import/extensions */
```

Comment:
Does this imply we should be updating our eslint config?

Reply:
Perhaps – I didn't atm just as we currently have quite a few of the same exemption in PIE, so I figure it's probably a wider conversation than doing git as part of this.